lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -1360,6 +1346,38 @@ def test_profile_image_requested_field_endorsed_comments(self):
             self.assertEqual(expected_author_profile_data, response_users[response_comment['author']])
             self.assertEqual(expected_endorser_profile_data, response_users[response_comment['endorsed_by']])
 
+    def test_profile_image_requested_field_null_endorsed(self):
```

Comment:
Name is inaccurate and difficult to read.  Maybe `test_profile_image_request_for_null_endorsed_by`?